opensrp-anc/src/main/java/org/smartregister/anc/library/util/ANCFormUtils.java

```diff
@@ -185,20 +185,6 @@ private static void processCheckBoxSpecialWidget(JSONObject widget, List<String>
         }
     }
 
-    public static JSONObject generateTranslatableValue(String value, JSONObject jsonObject) throws JSONException {
-        JSONObject newValue = new JSONObject();
-        ANCFormUtils formUtils = new ANCFormUtils();
-        if (jsonObject.has(JsonFormConstants.OPTIONS_FIELD_NAME)) {
-            JSONArray options = jsonObject.getJSONArray(JsonFormConstants.OPTIONS_FIELD_NAME);
-            JSONObject selectedOption = formUtils.getOptionFromOptionsUsingKey(options, value);
-            newValue.put(JsonFormConstants.VALUE, value);
-            newValue.put(JsonFormConstants.TEXT, selectedOption.optString(JsonFormConstants.TRANSLATION_TEXT, ""));
-            return newValue;
-        }
-        newValue.put(JsonFormConstants.VALUE, value);
-        newValue.put(JsonFormConstants.TEXT, jsonObject.optString(JsonFormConstants.TRANSLATION_TEXT, ""));
-        return newValue;
-    }
```

Comment:
Any reason this was moved from one Utils file to another?

Reply:
None. Just saw it looked better under Utils section

Reply:
This was done